datasets/wikimedia_translations/README.md

```diff
@@ -0,0 +1,428 @@
+- gu
+- gv
+- ha
+- hak
+- haw
+- he
+- hi
+- hif
+- hr
+- hsb
+- ht
+- hu
+- hy
+- hyw
+- ia
+- id
+- ie
+- ig
+- ik
+- ilo
+- inh
+- io
+- is
+- it
+- iu
+- ja
+- jam
+- jbo
+- jv
+- ka
+- kaa
+- kab
+- kbd
+- kg
+- ki
+- kk
+- kl
+- km
+- kn
+- ko
+- krc
+- ks
+- ksh
+- ku
+- kv
+- kw
+- ky
+- la
+- lad
+- lb
+- lbe
+- lez
+- lfn
+- lg
+- li
+- lij
+- lmo
+- ln
+- lo
+- lrc
+- lt
+- ltg
+- lv
+- lzh
+- mai
+- map_bms
+- mdf
+- mg
+- mhr
+- mi
+- min
+- mk
+- ml
+- mn
+- mr
+- mrj
+- ms
+- mt
+- mwl
+- my
+- myv
+- mzn
+- na
+- nah
+- nan
+- nap
+- nb
+- nds
+- nds_nl
+- ne
+- new
+- nl
+- nn
+- no
+- nov
+- nrm
+- nso
+- nv
+- ny
+- oc
+- olo
+- om
+- or
+- os
+- pa
+- pag
+- pam
+- pap
+- pcd
+- pdc
+- pfl
+- pi
+- pih
+- pl
+- pms
+- pnb
+- pnt
+- ps
+- pt
+- qu
+- rm
+- rmy
+- ro
+- ru
+- rue
+- rup
+- rw
+- sa
+- sah
+- sat
+- sc
+- scn
+- sco
+- sd
+- se
+- sgs
+- sh
+- si
+- simple
+- sk
+- sl
+- sm
+- sn
+- so
+- sq
+- sr
+- srn
+- ss
+- st
+- stq
+- su
+- sv
+- sw
+- szl
+- ta
+- tcy
+- te
+- tet
+- tg
+- th
+- ti
+- tk
+- tl
+- tn
+- to
+- tpi
+- tr
+- ts
+- tt
+- tum
+- tw
+- ty
+- tyv
+- udm
+- ug
+- uk
+- ur
+- uz
+- ve
+- vec
+- vep
+- vi
+- vls
+- vo
+- vro
+- wa
+- war
+- wuu
+- xal
+- xh
+- xmf
+- yi
+- yue
+- za
+- zea
+- zh
+- zh_tw
+- zh_yue
+- zu
+licenses:
+- unknown
+multilinguality:
+- multilingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+task_ids:
+- machine-translation
+---
+
+# Dataset Card Creation Guide
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** http://opus.nlpl.eu/wikimedia.php
+- **Repository:** None
+- **Paper:** http://www.lrec-conf.org/proceedings/lrec2012/pdf/463_Paper.pdf
+- **Leaderboard:** [More Information Needed]
+- **Point of Contact:** [More Information Needed]
+
+### Dataset Summary
+
+[More Information Needed]
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
```

Comment:
can you mention how to load the other languages like you did in other PRs please ?